test/profile_eos.cpp

```diff
@@ -222,8 +222,8 @@ int main(int argc, char *argv[]) {
 
     EOSBuilder::EOSType type = EOSBuilder::EOSType::IdealGas;
     EOSBuilder::params_t params;
-    params["Cv"] = 1e7 * 0.716; // specific heat in ergs/(g K)
-    params["gm1"] = 0.4;        // gamma - 1
+    params["Cv"].emplace<Real>(1e7 * 0.716); // specific heat in ergs/(g K)
+    params["gm1"].emplace<Real>(0.4);        // gamma - 1
```

Comment:
This was actually an error having to do with variant.

Reply:
Assignment requires that C++ can unambiguously cast the type into the types allowed in the variant. Type inferrence was failing at that cast. `emplace<Real>` lets me explicitly tell the compiler "I want this to be stored as a `Real` in the variant."

Reply:
Thank you! That makes sense